balancer-js/src/modules/data/token-prices/aave-rates.ts

```diff
@@ -0,0 +1,45 @@
+import { Interface } from '@ethersproject/abi';
+import { Contract } from '@ethersproject/contracts';
+import { Provider } from '@ethersproject/providers';
+import { formatUnits } from '@ethersproject/units';
+import { Multicall } from '@/modules/contracts/multicall';
+import { yieldTokens } from '../token-yields/tokens/aave';
+
+const wrappedATokenInterface = new Interface([
+  'function rate() view returns (uint256)',
+]);
+
+export class AaveRates {
+  multicall: Contract;
+  rates?: Promise<{ [wrappedATokenAddress: string]: number }>;
+
+  constructor(multicallAddress: string, provider: Provider) {
+    this.multicall = Multicall(multicallAddress, provider);
+  }
+
+  private async fetch(): Promise<{ [wrappedATokenAddress: string]: number }> {
+    console.time('Fetching aave rates');
```

Comment:
Can we use debug for these `console.time`'s so they can be enabled/disabled as they fill up the API logs. Could be done in a future PR.

Reply:
are you using a package in the API? all console.time / log etc. gets removed in the build step.